java.d/src/main/java/org/firehol/netdata/module/jmx/configuration/JmxQueryConfiguration.java

```diff
@@ -0,0 +1,27 @@
+package org.firehol.netdata.module.jmx.configuration;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+/**
+ * Configuration scheme of a JMX property queried by the
+ * {@link org.firehol.netdata.module.jmx.JmxModule}.
+ */
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+public class JmxQueryConfiguration {
```

Comment:
I think we should support compositeDataKey for every JmxQuery if a generic implementation is possible.